tests/python/pants_test/init/test_logging.py

```diff
@@ -7,10 +7,9 @@
 import logging
 import unittest
 import uuid
-from builtins import str
+from builtins import open, str
 from contextlib import closing, contextmanager
-
-import six
+from io import StringIO
```

Comment:
`six.StringIO` is not safe. It has different semantics in Py2. We'll need to strip all remaining instances.